lib/recaptcha/client_helper.rb

```diff
@@ -84,5 +85,52 @@ def self.recaptcha_components(options = {})
 
       [html, tag_attributes, fallback_uri]
     end
+
+    private
+
+    def default_callback_name
+      'invisibleRecaptchaSubmit'
+    end
+
+    def default_callback_script
+      <<-HTML
+        <script>
+          var createClosestPolyfill = function () {
```

Comment:
not a fan of adding global methods ...

Reply:
Noted. I'll look into nesting this under the default callback function.